src/OpenTelemetry.Instrumentation.AspNetCore/Implementation/HttpInListener.cs

```diff
@@ -227,5 +234,30 @@ private static string GetUri(HttpRequest request)
 
             return builder.ToString();
         }
+
+        private bool TryGetGrpcMethod(Activity activity, out string grpcMethod)
+        {
+            grpcMethod = Dependencies.GrpcTagHelper.GetGrpcMethodFromActivity(activity);
+            return !string.IsNullOrEmpty(grpcMethod);
+        }
+
+        private void AddGrpcAttributes(Activity activity, string grpcMethod, HttpContext context)
+        {
+            // TODO: Should the leading slash be trimmed? Spec seems to suggest no leading slash: https://github.com/open-telemetry/opentelemetry-specification/blob/master/specification/trace/semantic_conventions/rpc.md#span-name
+            // Client instrumentation is trimming the leading slash. Whatever we decide here, should we apply the same to the client side?
+            // activity.DisplayName = grpcMethod?.Trim('/');
```

Comment:
While this seems to be what the spec suggest, do we really want to trim the leading slash? It is what I did with the gRPC client instrumentation. Though, doing this here does effectively rename the activity as generated by ASP.NET Core.